src/traffic_server/InkIOCoreAPI.cc

```diff
@@ -634,11 +634,12 @@ TSIOBufferWrite(TSIOBuffer bufp, const void *buf, int64_t length)
 }
 
 // not in SDK3.0
```

Comment:
Is this a valid comment?

Reply:
I don't know. What's "SDK 3.0" anymore? We took out the version checking.